packages/graphql-transformers-e2e-tests/src/__tests__/AuthV2Transformer.e2e.test.ts

```diff
@@ -22,15 +22,18 @@ import {
 } from '../cognitoUtils';
 import { ResourceConstants } from 'graphql-transformer-common';
 import { GraphQLClient } from '../GraphQLClient';
+import { resolveTestRegion } from '../testSetup';
+
+const region = resolveTestRegion();
```

Comment:
Can we keep the region variable name same across all the tests? Currently, there are three references 'REGION', 'region' and 'AWS_REGION'.